osgp/shared/osgp-ws-smartmetering/src/main/resources/SmartMeteringConfiguration.wsdl

```diff
@@ -709,698 +733,729 @@
-        <soap:body use="literal"/>
+          use="literal" />
+        <soap:body use="literal" />
       </wsdl:input>
       <wsdl:output name="ConfigureDefinableLoadProfileAsyncResponse">
-        <soap:body use="literal"/>
+        <soap:body use="literal" />
       </wsdl:output>
     </wsdl:operation>
     <wsdl:operation name="GetConfigureDefinableLoadProfileResponse">
-      <soap:operation soapAction=""/>
+      <soap:operation soapAction="" />
       <wsdl:input name="ConfigureDefinableLoadProfileAsyncRequest">
         <soap:header message="tns:ActionHeader" part="OrganisationIdentification"
-                     use="literal"/>
+          use="literal" />
         <soap:header message="tns:ActionHeader" part="UserName"
-                     use="literal"/>
+          use="literal" />
         <soap:header message="tns:ActionHeader" part="ApplicationName"
-                     use="literal"/>
-        <soap:body use="literal"/>
+          use="literal" />
+        <soap:body use="literal" />
       </wsdl:input>
       <wsdl:output name="ConfigureDefinableLoadProfileResponse">
-        <soap:body use="literal"/>
+        <soap:body use="literal" />
       </wsdl:output>
     </wsdl:operation>
 
     <wsdl:operation name="SetMbusUserKeyByChannel">
-      <soap:operation soapAction=""/>
+      <soap:operation soapAction="" />
       <wsdl:input name="SetMbusUserKeyByChannelRequest">
         <soap:header message="tns:ActionHeader" part="OrganisationIdentification"
-                     use="literal"/>
+          use="literal" />
         <soap:header message="tns:ActionHeader" part="UserName"
-                     use="literal"/>
+          use="literal" />
         <soap:header message="tns:ActionHeader" part="ApplicationName"
-                     use="literal"/>
-        <soap:body use="literal"/>
+          use="literal" />
+        <soap:body use="literal" />
       </wsdl:input>
       <wsdl:output name="SetMbusUserKeyByChannelAsyncResponse">
-        <soap:body use="literal"/>
+        <soap:body use="literal" />
       </wsdl:output>
     </wsdl:operation>
 
     <wsdl:operation name="GetSetMbusUserKeyByChannelResponse">
-      <soap:operation soapAction=""/>
+      <soap:operation soapAction="" />
       <wsdl:input name="SetMbusUserKeyByChannelAsyncRequest">
         <soap:header message="tns:ActionHeader" part="OrganisationIdentification"
-                     use="literal"/>
+          use="literal" />
         <soap:header message="tns:ActionHeader" part="UserName"
-                     use="literal"/>
+          use="literal" />
         <soap:header message="tns:ActionHeader" part="ApplicationName"
-                     use="literal"/>
+          use="literal" />
       </wsdl:input>
       <wsdl:output name="SetMbusUserKeyByChannelResponse">
-        <soap:body use="literal"/>
+        <soap:body use="literal" />
+      </wsdl:output>
+    </wsdl:operation>
+
+    <wsdl:operation name="GetUnencryptedKeys">
```

Comment:
check, ik zal eens kijken of ik WSDL validatie kan aanzetten in IntelliJ.

Reply:
Keep the conversation in English please.

Enabling WSDL validation has not been entirely successful yet, as far as I can tell.
`name="GetKeysAsyncResponse"` on line 448 is an invalid value, since in `sm-configuration.xsd` renames were only partially executed, and there is `GetUnencryptedKeysAsyncResponse` instead of `GetKeysAsyncResponse`.